semantic-conventions/src/opentelemetry/semconv/model/semantic_attribute.py

```diff
@@ -117,7 +117,10 @@ def parse(
             if attr_id is not None:
                 validate_id(attr_id, position_data["id"])
                 attr_type, brief, examples = SemanticAttribute.parse_id(attribute)
-                fqn = "{}.{}".format(prefix, attr_id)
+                if prefix:
```

Comment:
I did not add this bugfix to the changelog because IMHO the prefix being optional is a misfeature, and we should require the prefix and/or default it to the semantic convention ID.